### ISSUE TITLE
update subsection: Structure-based prediction of bioactivity

### ISSUE THREAD

**Issue by 0ut0fcontrol** [greenelab/deep-review#1011]
Hi,
@mapleleaf-soar and I found AI models (e.g. AtomNet #56, ACNN #287) achieve high performance on DUD-E and PDBbind data sets because they learned the bias in the data sets. REF #1008 

For example:
> We found ACNN  model can achieve "state-of-the-art" in PDBbind data set using **ligand alone**.😂
![acnn](https://user-images.githubusercontent.com/11703338/78219681-98a3ca00-74f2-11ea-94f0-1fc7fba8107a.png)

Others also found the bias in DUD-E. REF #1009 #1010

I believe the subsection `Structure-based prediction of bioactivity` need to be updated and try to work on it.

- The impact of deep learning in treating disease and developing new treatments
    - Drug development
        - [Structure-based prediction of bioactivity](https://github.com/greenelab/deep-review/blob/master/content/05.treat.md#structure-based-prediction-of-bioactivity)

Any comments and suggestions are welcome!

**Comment by agitter** [comment]
Thanks for the suggestions.  Would you like to write a short update to this section?  Discussing bias in data sets and evaluations is definitely of interest to us.

I could likely review that pull request, though I can't promise a specific timeline.  If you want to make a pull request, I recommend
- Skimming #988 to make sure your edits won't introduce conflicts.  If so, let's try to merge that one first.
- Being explicit about which other models discussed in the review are affected by this dataset bias.

**Comment by cgreene** [comment]
I agree with @agitter : a short update to that section as a pull request would be great - thanks - with the same caveats :)
